examples/rcbilling-demo/src/pages/login/index.tsx

```diff
@@ -16,9 +26,13 @@ const LoginPage: React.FC = () => {
             const appUserId = (
               document.getElementById("app-user-id") as HTMLInputElement | null
             )?.value;
-            if (appUserId) {
-              navigate(`/paywall/${encodeURIComponent(appUserId)}`);
-            }
+            navigateToAppUserIDPaywall(appUserId);
+          }}
+        />
+        <Button
+          caption="Skip"
```

Comment:
I decided to add this skip button to make it simpler to test... But lmk if you would prefer something less visible in the demo app.